modules/data/event-hub/main.bicep

```
@@ -144,7 +144,7 @@ resource eventHubNamespace 'Microsoft.EventHub/namespaces@2021-11-01' = [for var
   tags: tags
 }]
 
-module eventHubNamespace_authorizationRules 'authorizationRules/deploy.bicep' = [for (varNamespaceAuthorizationRule, index) in varNamespaceAuthorizationRules: if (!empty(namespaceAuthorizationRules)) {
+module eventHubNamespace_authorizationRules 'modules/authorizationRules.bicep' = [for (varNamespaceAuthorizationRule, index) in varNamespaceAuthorizationRules: if (!empty(namespaceAuthorizationRules)) {
```

```suggestion
module eventHubNamespace_authorizationRules 'modules/authorizationRules.bicep' = [for (varNamespaceAuthorizationRule, index) in varNamespaceAuthorizationRules: {
```

don't need to check if group is empty, this is automatically handled by the for-in logic.